before_you_start.md

```diff
@@ -1,6 +1,14 @@
 ## Before you start
+
+### Testing SSH
+If you haven't yet done so, you should make sure you have created SSH keys and are able to use them to connect to a remote machine.
+
+Github has a comprehensive guide for all major operating systems: [Connecting to Github with SSH](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh)
+
+## Optional setup for the afternoon (required for the project exercise)
+
 ### Hypervisor Setup
-For this workshop you will need a hypervisor installed on your development machine. If you have Windows Hyper-V is preferred; otherwise Oracle VirtualBox is a free cross-platform alternative.
+For the afternoon exericse, you will need a hypervisor installed on your development machine. Oracle VirtualBox is a free cross-platform hypervisor, or if you are on Windows Hyper-V is also an option.
```

Comment:
```suggestion
For the afternoon exercise, you will need a hypervisor installed on your development machine. Oracle VirtualBox is a free cross-platform hypervisor, or if you are on Windows Hyper-V is also an option.
```

Reply:
If we're telling them it's optional in the header then should this sentence clarify that by saying something like "for the optional vagrant exercise in the afternoon, you would need ..."